### PR TITLE
Support 'request response lite' mode

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: src/templates/_helpers.tpl [hunk 1]
```diff
@@ -175,6 +175,8 @@ http://{{ include "apicurio-registry.fullname" ( index .Subcharts "apicurio-regi
 nussknacker-scenario
 {{- else if eq .Values.nussknacker.mode "streaming-lite" -}}
 nussknacker-lite-scenario
+{{- else if eq .Values.nussknacker.mode "request-response-lite" -}}
+nussknacker-request-response-lite-scenario
```
Comment (arkadius) [inline src/templates/_helpers.tpl:179]: We don't have such dashboard yet. Do we?

**Review by arkadius — COMMENTED** [review]
Nice, some comments added

File: build-deploy-verify.sh [hunk 2]
```diff
@@ -12,6 +12,8 @@ shift
 cd "$(dirname "$0")" && rm -rf dist/
 helm package -d dist/ src/
 kubectl get secret "$RELEASE-postgresql" || cat postgres-secret.yaml | POSTGRES_PASSWORD=`date +%s | sha256sum | base64 | head -c 32` RELEASE=$RELEASE MAYBE_NAMESPACE=`kubectl config view --minify -o jsonpath='{..namespace}'` NAMESPACE=${MAYBE_NAMESPACE:-default} envsubst | kubectl apply -f -
+
+sleep 5
```
[the remark this answers is not in the document]
Comment (gadomsky) [reply build-deploy-verify.sh]: I came across wierd behaviour, when secret was just created. Deployments which was spawned right after that, by helm, like didnt see it? I dont know, cause now I cannot reprudce it.. 

Deleted.

File: .github/workflows/helm-test-workflow.yaml [hunk 3]
```diff
@@ -86,14 +86,35 @@ jobs:
             --config=.k3d/single-cluster.yml
             --trace
       - name: helmUpgrade
-        run: helm upgrade -i "${{env.RELEASE_NAME}}" dist/*.tgz --version "${{needs.build.outputs.version}}" --wait --debug -f deploy-values.yaml -f deploy-values-lite.yaml
+        run: helm upgrade -i "${{env.RELEASE_NAME}}" dist/*.tgz --version "${{needs.build.outputs.version}}" --wait --debug -f deploy-values.yaml -f deploy-values-kafka-config.yaml -f deploy-values-streaming-lite.yaml
+      - name: helmTestRelease
+        run: ./test.sh "${{env.RELEASE_NAME}}"
+  test-request-response-lite:
+    name: test streaming-request-response mode
```
[the remark this answers is not in the document]
Comment (gadomsky) [reply .github/workflows/helm-test-workflow.yaml]: fixed, thanks

File: src/templates/_helpers.tpl (hunk 1 above)
Comment (arkadius) [reply src/templates/_helpers.tpl:179]: In our dev config I've added for now nussknacker-lite-scenario as a starting point for tests. But can it be that way as well

**Review by arkadius — APPROVED** [review]

**Pull request merged**